### ISSUE TITLE
Problem using two Trix editors on the same form.

### ISSUE THREAD

**Issue by andrewculver** [kylefox/trix#14]
**Describe the bug**
When I include two Trix editors for two different fields in the same form, it actually just repeats the first field a second time.

**To Reproduce**
1. Create a model with two separate `text` columns.
2. Scaffold some views and a controller.
3. Use the scaffolding to create a new copy of that model with a different value in each column.
4. Update the form to use `trix_editor` instead of `text_area` and reload the edit page.
5. Observe that it's duplicating the content of the first field.

**Expected behavior**
It should be two different fields.

**Actual behavior**
It's the first field duplicated.

**Screenshots or Video**

This is using `text_area`:

<img width="828" alt="screenshot 2018-11-15 19 30 21" src="https://user-images.githubusercontent.com/217218/48594739-3dbd1180-e995-11e8-8c0b-591ae774692d.png">

This is using `trix_editor`:

<img width="833" alt="screenshot 2018-11-15 19 31 30" src="https://user-images.githubusercontent.com/217218/48594749-4a416a00-e995-11e8-93a6-51147ddd9616.png">

(Both fields are `trix_editor`, we're just hiding the toolbar on the first one because it's not in focus.)

**Environment (please complete the following information):**
 - Rails version: 5.2.1
 - trix-rails version: 2.0.0
- Browser: Safari 11.1 (13605.1.33.1.2)

**Comment by andrewculver** [comment]
@kylefox This is an issue we ran into while integrating Trix into Bullet Train. I'm having a developer take a look at this on our end, I just wanted to document it and submit the PR in the right place.

**Comment by kylefox** [comment]
@andrewculver Thanks for reporting, good catch!

**Comment by kylefox** [comment]
@andrewculver I just setup a [scaffold project](https://github.com/kylefox/trix-rails-issue-14) as you described, and I'm not able to reproduce the problem. Maybe I missed something?

![trix](https://user-images.githubusercontent.com/37838/48597997-e3349e80-e91d-11e8-826f-8a01788b8479.gif)

**Comment by andrewculver** [comment]
@kylefox Huh. I'm seeing the same thing as you (e.g. correct behavior) when I spin up an entirely new app and scaffold a new model. I'll report back if/when we figure out why this is happening in our own app. Sorry to waste your time!

**Comment by kylefox** [comment]
@andrewculver No problem — are you using either the Formtastic or Simple Form helper?

**Comment by andrewculver** [comment]
@kylefox Nope. I was surprised when another developer on the team said they couldn't reproduce the issue either, so I checked out a fresh copy of the repository and it's working totally fine. I'm clueless, but something bad must have gotten cached. Appreciate again the time you spent on this!

**Comment by andrewculver** [comment]
@kylefox No, we don't use either of those, but I did just confirm that the problem is with some form helpers we provide. Sorry again, but really appreciate the time you spent on this! 🙏

**Comment by andrewculver** [comment]
@kylefox I spoke a little too soon and I actually figured out what the issue is here. I struggled to reproduce this reliably, but for some reason when Rails `form_with` wasn't generating IDs for labels and fields by default (e.g. Rails 5.0 and maybe 5.1,) it caused this situation because the different Trix editor fields were ending up with the same HTML element ID (e.g. only the latter part of the ID as generated, not the earlier part that has the field name in it.) 🤷‍♂️ The fix was, in a Rails 5.2 app, to make sure this configuration setting was present in `config/application.rb`:

```
    # Initialize configuration defaults for originally generated Rails version.
    config.load_defaults 5.2
```

Which enables the ID generation for form fields and labels by default. Alternatively, if you don't want all the 5.2 defaults, someone could just do: `config.action_view.form_with_generates_ids = true` and that would also fix this problem.

Just documenting this here incase anyone else runs into this totally weird situation, although I assume it won't come up very often going forward since ID generation for HTML labels and fields has been re-enabled in Rails by default going forward.
